resources/fixtures/Robe/Robe-Robin-300E-Beam.qxf

```diff
@@ -111,33 +119,42 @@
  </Channel>
  <Channel Name="Static Gobo Wheel">
   <Group Byte="0">Gobo</Group>
-  <Capability Min="0" Max="0" Preset="GoboMacro" Res1="Others/open.svg">Continual positioning open/hole (6.5°)</Capability>
-  <Capability Min="7" Max="7" Preset="GoboMacro" Res1="SGM/gobo00061.svg">Continual positioning beam reducer (5.5)</Capability>
-  <Capability Min="14" Max="14" Preset="GoboMacro" Res1="SGM/gobo00062.svg">Continual positioning beam reducer (4.5)</Capability>
-  <Capability Min="21" Max="21" Preset="GoboMacro" Res1="SGM/gobo00063.svg">Continual positioning beam reducer (3)</Capability>
-  <Capability Min="28" Max="28" Preset="GoboMacro" Res1="SGM/gobo00064.svg">Continual positioning beam reducer (1.5)</Capability>
-  <Capability Min="35" Max="35" Preset="GoboMacro" Res1="Robe/gobo00070.svg">Continual positioning 1</Capability>
-  <Capability Min="42" Max="42" Preset="GoboMacro" Res1="Robe/gobo00027.svg">Continual positioning 2</Capability>
-  <Capability Min="49" Max="49" Preset="GoboMacro" Res1="Robe/gobo00026.svg">Continual positioning 3</Capability>
-  <Capability Min="56" Max="56" Preset="GoboMacro" Res1="Robe/gobo00025.svg">Continual positioning 4</Capability>
-  <Capability Min="64" Max="64" Preset="GoboMacro" Res1="Others/open.svg">Continual positioning open/hole</Capability>
-  <Capability Min="65" Max="70" Preset="GoboMacro" Res1="SGM/gobo00061.svg">Positioning beam reducer (5.5°)</Capability>
-  <Capability Min="71" Max="75" Preset="GoboMacro" Res1="SGM/gobo00062.svg">Positioning beam reducer (4.5°)</Capability>
-  <Capability Min="76" Max="81" Preset="GoboMacro" Res1="SGM/gobo00063.svg">Positioning beam reducer (3°)</Capability>
-  <Capability Min="82" Max="86" Preset="GoboMacro" Res1="SGM/gobo00064.svg">Positioning beam reducer (1.5°)</Capability>
-  <Capability Min="87" Max="92" Preset="GoboMacro" Res1="Robe/gobo00070.svg">Positioning 1</Capability>
-  <Capability Min="93" Max="97" Preset="GoboMacro" Res1="Robe/gobo00027.svg">Positioning 2</Capability>
-  <Capability Min="98" Max="103" Preset="GoboMacro" Res1="Robe/gobo00026.svg">Positioning 3</Capability>
-  <Capability Min="104" Max="109" Preset="GoboMacro" Res1="Robe/gobo00025.svg">Positioning 4</Capability>
+  <Capability Min="0" Max="0" Preset="GoboMacro" Res1="Others/open.svg">Open/hole (6.5°)</Capability>
+  <Capability Min="1" Max="6">Open/Beam reducer (5.5)</Capability>
```

Comment:
I'm not really sure how this could be treated.
Intermediate positions are quite useless in most cases.
I think this should be "0-6" open, "7-13" reducer and so on.
Then QLC+ should handle the intermediate positions

Reply:
For intermediate color positions, I see the benefit, but for the intermediate gobo and beam reducer positions, I also see no use for them (they say fine positioning - so moving a cropped beam around in the full beam window), except maybe for very rare and special cases. Nevertheless, the manufacturers provided these as a separate feature (the fixed positions with DMX ranges are there, too), so I think, we will need to represent them.
https://www.robe.cz/res/downloads/dmx_charts/Robin_300E_Beam_DMX_charts.pdf

If so, we would need to make space around the full positions, but I wouldn't go so far:
0-4: Open 6.5°
5-12: Reducer 5.5°
...
61-64: Open 6.5°

Reply:
Same as my comment above. Let's leave these without preset for now.